src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/SecurityExtensionTest.php

```diff
@@ -900,6 +900,28 @@ public function testCustomHasherWithMigrateFrom()
         ]);
     }
 
+    public function testMigrateToLegacyHasherThrows()
```

Comment:
Could we add a data provider and name the forbidden ones? Can be done after decision on this PR